zeebe/benchmarks/project/src/main/java/io/camunda/zeebe/OperateClient.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.util.CloseableSilently;
+import java.io.IOException;
+import java.net.Authenticator;
+import java.net.PasswordAuthentication;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Version;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import org.agrona.CloseHelper;
+
+final class OperateClient implements CloseableSilently {
```

Comment:
🤡 While we get rid of the internal API and V1, I see more an more us creating such clients in different places. I feel there is a need to have some common lib providing such (tbh wondering why this doesn't exist already)

Other example https://github.com/camunda/camunda/blob/main/qa/util/src/main/java/io/camunda/qa/util/cluster/TestRestOperateClient.java